src/driver.rs

```diff
@@ -163,33 +166,7 @@ impl rustc_driver::Callbacks for ClippyCallbacks {
 }
 
 fn display_help() {
-    println!(
-        "\
-Checks a package to catch common mistakes and improve your Rust code.
-
-Usage:
-    cargo clippy [options] [--] [<opts>...]
-
-Common options:
-    -h, --help               Print this message
-        --rustc              Pass all args to rustc
-    -V, --version            Print version info and exit
-
-For the other options see `cargo check --help`.
-
-To allow or deny a lint from the command line you can use `cargo clippy --`
-with:
-
-    -W --warn OPT       Set lint warnings
-    -A --allow OPT      Set lint allowed
-    -D --deny OPT       Set lint denied
-    -F --forbid OPT     Set lint forbidden
-
-You can use tool lints to allow or deny lints from your code, eg.:
-
-    #[allow(clippy::needless_lifetimes)]
-"
-    );
+    println!("{}", help_message());
```

Comment:
```rust
#[allow(clippy::ignored_unit_patterns)]
fn display_help() {
    println!("{}", help_message());
}
```

You can put it on the `display_help` function directly, which only has this one single macro call in it.

FYI, this would also work, but I prefer the former:
```rust
fn display_help() {
    #![allow(clippy::ignored_unit_patterns)]
    println!("{}", help_message());
}
```